Pipeline/D - Model sensitivity analyses and checks/D03 - Additional robustness.R

```diff
@@ -254,16 +243,318 @@ for(m in 1:length(modellist)) {
     polyOrder = 2, cluster = T, xRef = myrefT,
     xLab = expression(paste("Mean temperature (",degree,"C)")), 
     yLab = "Prevalence (%)", title = mycollabs[m], yLim=c(-30,10), showYTitle = T) +
-    theme(plot.title = element_text(size = 10))
+    theme(plot.title = element_text(size = 10)) +
+    geom_vline(xintercept = percentiles_list[[m]]$p01, colour = "grey39", linetype = "dashed") +
+    geom_vline(xintercept = percentiles_list[[m]]$p99, colour = "grey39", linetype = "dashed")
 }
 
-p = plot_grid(figList[[1]], figList[[2]], nrow=1)
-p
+h_pre <- ggplot(data = pre_data, aes(x = temp)) +
+  geom_histogram(
+    fill = "#8B3A4A",
+    alpha = 1,
+    bins = 30,
+    width = 0.7,
+    colour = "black"
+  ) +
+  theme_classic() +
+  labs(x = expression(paste("Mean temperature (",degree,"C)")), y = NULL) +
+  geom_vline(xintercept = percentiles_list[[1]]$p01, colour = "grey39", linetype = "dashed") +
+  geom_vline(xintercept = percentiles_list[[1]]$p99, colour = "grey39", linetype = "dashed") +
+  annotate(
+    geom = "text", x = 36, y = 0, vjust = -1, 
+    label = bquote(italic("N")~"="~.(percentiles_list[[1]]$n)),
+    size = 3
+  ) +
+  xlim(Tmin - .0001, Tmax) +
+    geom_vline(xintercept = 22, colour = "grey39") +
+  theme(
+    axis.text.y = element_blank(),
+    axis.ticks.y = element_blank(),
+    plot.margin = unit(c(-2.5, 1, 0, 1), "cm"),
+  )
+
+h_post <- ggplot(data = pos_data, aes(x = temp)) +
+  geom_histogram(
+    fill = "#8B3A4A",
+    alpha = 1,
+    bins = 30,
+    width = 0.7,
+    colour = "black"
+  ) +
+  theme_classic() +
+  labs(x = expression(paste("Mean temperature (",degree,"C)")), y = NULL) +
+  xlim(Tmin - .0001, Tmax) +
+  geom_vline(xintercept = 25, colour = "grey39") +
+  geom_vline(xintercept = percentiles_list[[2]]$p01, colour = "grey39", linetype = "dashed") +
+  geom_vline(xintercept = percentiles_list[[2]]$p99, colour = "grey39", linetype = "dashed") +
+  annotate(
+    geom = "text", x = 36, y = 0, vjust = -1, 
+    label = bquote(italic("N")~"="~.(percentiles_list[[2]]$n)),
+    size = 3
+  ) +
+  theme(
+    axis.text.y = element_blank(),
+    axis.ticks.y = element_blank(),
+    plot.margin = unit(c(-2.5, 1, 0, 1), "cm"),
+  )
+
+p = plot_grid(
+  figList[[1]] + 
+    theme(
+      axis.text.x = element_blank(),
+      axis.line.x = element_blank(),
+      axis.ticks.x = element_blank(),
+      axis.title.x = element_blank()
+    ),
+  figList[[2]] +
+    theme(
+      axis.text.x = element_blank(),
+      axis.line.x = element_blank(),
+      axis.ticks.x = element_blank(),
+      axis.title.x = element_blank()
+    ), 
+  h_pre, 
+  h_post,  
+  nrow=2,
+  align = "v",
+  rel_heights = c(15, 1)
+)
+# p
+
+ggsave(
+  filename = "split_sample_1995.pdf",
+  # path = file.path(resdir, "Figures", "Diagnostics", "Subsamples"),
+  plot = p, 
+  width = 10, 
+  height = 5
+)
+
+########################################################################
+# Data imbalance: responses on spatial subsamples ----
```

Comment:
@cullen-molitor this is a flag that i'm not yet sure if we are keeping this analysis or not. once the response is finalized we may want to come remove this just so it's not confusing